spec/uktt_api_spec.rb

```diff
@@ -444,4 +444,20 @@
       expect(response[:data].first[:attributes][:quota_order_number_id]).to eq(quota_search_params[:order_number])
     end
   end
+
+  it 'performs a retreive of countries' do
+    Uktt.configure(format: 'jsonapi', version: 'v2')
+
+    expect_any_instance_of(Uktt::Http).to receive(:retrieve).with('geographical_areas/countries', 'jsonapi')
+
+    Uktt::Country.new.retrieve
+  end
+
+  it 'performs a retreive of geographical_areas' do
+    Uktt.configure(format: 'jsonapi', version: 'v2')
+
+    expect_any_instance_of(Uktt::Http).to receive(:retrieve).with('geographical_areas', 'jsonapi')
```

Comment:
Hmm, I think that if you add a spy on UkTT::Http, you can just say

```ruby
expect(Uktt::Http).to have_received(:retrieve).with('geographical_areas', 'jsonapi')
```

Here?

Reply:
Is an instance